main.go

```diff
@@ -8,5 +8,23 @@ func main() {
 	// this is no purpose comment and should trigger a review comment
 	// fmt.Prinln("hello world")
 
+	// new comment
+
+	// comment bizare
+
+	x := 40
+	if x == 40 {
+		fmt.Println("x is 40")
+	} else {
+		fmt.Println("x is not 40")
+	}
+
+	secret := "my secret"
+	if secret == "my secret" {
+		fmt.Println("secret is my secret")
```

Comment:
The 'if' statement checks if 'secret' is equal to 'my secret', but 'secret' is not defined anywhere else in the code. This condition will always be true, and the 'else' branch should be removed.